jobs/nats-tls/templates/nats-tls.conf.erb

```diff
@@ -24,12 +24,16 @@
   nats_peers = nil
   nats_user = nil
   nats_password = nil
+  nats_auth_required = true
   if_p("nats.user") do |user|
     nats_user = user
   end
   if_p("nats.password") do |password|
     nats_password = password
   end
+  if_p("nats.auth_required") do |auth_required|
+    nats_auth_required = auth_required
+  end
```

Comment:
Done!

Reply:
this works, because `nats.auth_required` has a default value in the spec. other values like `nats.user` don't have one, that's why we need to check for their existence first.